### PR TITLE
uv, os: implement os.{get,set}Priority functions

### PR DESCRIPTION
**Pull request by yorkie**
This implements the following APIs:

- https://github.com/nodejs/node/blob/master/doc/api/os.md#osgetprioritypid
- https://github.com/nodejs/node/blob/master/doc/api/os.md#ossetprioritypid

With the group of APIs, we are able to tweak the priority level on demand at runtime.

##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added

### CODE REVIEW

File: src/js/os.js [hunk 1]
```diff
@@ -16,6 +16,41 @@ exports.freemem = function() {
   return native.getFreeMem();
 };
 
+exports.getPriority = function(pid) {
+  if (pid === undefined)
```
Comment (lolBig) [inline src/js/os.js]:
```suggestion
  if (pid === undefined) {
```

Comment (lolBig) [reply src/js/os.js]: missing `{`

File: src/js/os.js [hunk 2]
```diff
@@ -16,6 +16,41 @@ exports.freemem = function() {
...
+    priority = pid;
+    pid = 0;
+  } else {
+    pid = Number(pid);
+  }
+
+  if (typeof priority !== 'number') {
+    throw new TypeError('priority must be a number');
+  }
+  if (priority > 19) {
```
Comment (lolBig) [inline src/js/os.js]: use `UV_PRIORITY_LOW` and `UV_PRIORITY_HIGHST` defined  from  UV instead of literal ?

Comment (yorkie) [reply src/js/os.js]: Actually those defines are for using at Windows, I still leave those on uv.h is to acquire consistence.

Comment (lolBig) [reply src/js/os.js]: seems those defines are using at Linux too, see setPriority [doc](https://linux.die.net/man/2/setpriority)

Comment (yorkie) [reply src/js/os.js]: Could you please reference the code/lines in details? Can not search any that defines from your link :(

**Comment by yorkie** [comment]
The CI seems green now, @lolBig @legendecas please see again :)

File: src/js/os.js [hunk 3]
```diff
@@ -16,6 +16,42 @@ exports.freemem = function() {
...
+    priority = 19;
+  } else if (priority < -20) {
+    priority = -20;
```
Comment (legendecas) [inline src/js/os.js]: Should an error been thrown instead of silently rounded?

File: src/js/os.js [hunk 4]
```diff
@@ -16,6 +18,42 @@ exports.freemem = function() {
...
+    pid = Number(pid);
+  }
+
+  if (typeof priority !== 'number') {
+    throw new TypeError('priority must be a number');
+  }
+  if (priority > constants.UV_PRIORITY_LOW ||
+    priority < constants.UV_PRIORITY_HIGHEST) {
+    throw new RangeError(
+      'The value of "priority" is out of range. It must be > -20 && < 19.');
```
Comment (lolBig) [inline src/js/os.js]: use constants instead of -20 or 19

**Pull request merged**